.github/CONTRIBUTING.md

```diff
@@ -37,15 +38,30 @@ PlaceholderAPI provides a feature to have expansions (separate jar files) for pl
 In those cases should you report the issue to the issue tracker of the expansion or plugin.
 
 ## Pull requests
-As an open source project are we welcoming all contributions to improve PlaceholderAPI, being it changes to its code like bug fixes or new features, or contributions to its documentation such as the [Wiki] or the Javadoc.
+As an open source project are we welcoming all contributions to improve PlaceholderAPI, being it changes to its code, or contributions to its documentation such as the [Wiki] or the Javadocs.
+
+**When contributing, make sure to both base of and target the mentioned branch. Pull requests targeting the wrong branch may get closed without a warning.**
```

Comment:
This sentence right here doesn't feel like it flows properly. "Make sure to both base of and target the mentioned branch". Is "of" supposed to be "off"?

Reply:
Idk... Not a native english speaker so I can't really tell which one is correct here...

In that sentence I basically want to say that the source and target branch should be the mentioned one.